src/skin/tooltips.cpp

```diff
@@ -783,7 +794,7 @@ void Tooltips::addStandardTooltips() {
 
     add("EffectSlot_enabled")
             << tr("Enable Effect")
-            << tr("This effect chain must also be assigned to a deck or other sound source to hear the effect.");
+            << tr("This effect unit must also be assigned to a deck or other sound source to hear the effect.");
```

Comment:
For the ``EffectSlot_enabled`` control the text is about effect *unit*, same as with ``EffectUnit_``controls.
My understanding was ``racks -> unit -> slot``, has it changed?

Reply:
rack: distinction in the code that makes no difference to users. IMO we should not mention it in user visible strings
chain: a group of effects processed in series
unit: the thing that a chain is loaded into